nixos/lib/test-driver/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3Packages
+, enableOCR ? false
+, qemu_pkg ? qemu_test
+, coreutils
+, imagemagick_light
+, libtiff
+, netpbm
+, qemu_test
+, socat
+, tesseract4
+, vde2
+}:
+
+let
+  ocrProg = tesseract4.override { enableLanguages = [ "eng" ]; };
+  imagemagick_tiff = imagemagick_light.override { inherit libtiff; };
+in
+
+with python3Packages;
```

Comment:
when using `with`, can you please try to reduce its scope as much as possible? this makes it easier to distinguish what comes from where. see also https://nix.dev/anti-patterns/language#with-attrset-expression

Reply:
reduced scope